### ISSUE TITLE
No "lambda_" in the definition of norm_tv()

### ISSUE THREAD

**Issue by jacquesdurden** [epfl-lts2/pyunlocbox#42]
Dear developers, it seems that the parameter ```lambda_``` has been forgotten from the definition of ```norm_tv``` (as visible from [here](https://github.com/epfl-lts2/pyunlocbox/blob/master/pyunlocbox/functions.py)). Is it programmed like this on purpose?

**Comment by nperraud** [comment]
Hi @jacquesdurden , 
I think this parameter is inherited from the class `norm`.
See:
https://github.com/epfl-lts2/pyunlocbox/blob/66181218a3268444983233720f0dc10392c15d5a/pyunlocbox/functions.py#L405

**Comment by jacquesdurden** [comment]
Yes you right but inheritance is not enough here. In particular, if you set it, conversely to other norms, it has no effect, see the screenshot below.
![7XeDCnQeDSgVqjbn](https://user-images.githubusercontent.com/55459825/168025573-55aadc10-59e2-4c6b-8b76-0eef44112aa6.png)

**Comment by nperraud** [comment]
Well I call this a bug. Thanks for spotting and reporting it.

**Comment by jacquesdurden** [comment]
I guess this amounts to simply inserting a "gamma=self.step*lambda_" at the right places in norm_tv's code

**Comment by nperraud** [comment]
yes, we have to replace T by gamma where gamma = self.lambda_ * T.
this also require to write some tests to be sure that everything is correct. I am not sure when I will have the time to do.
this.
If you want, you are welcome to submit a PR. :-)
